XPlat/AzureMessaging/Android/samples/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="BuildOutput" value="../../output" />
```

Comment:
If you are using project references, then you shouldn't need this.

Reply:
Kept packageref, so keeping this